executor-service/src/main/java/executor/service/config/AppConfig.java

```diff
@@ -1,11 +1,26 @@
 package executor.service.config;
 
+import com.fasterxml.jackson.databind.ObjectMapper;
+import executor.service.facade.parallel.ConfigurableThreadPool;
 import executor.service.model.ThreadPoolConfig;
 import executor.service.model.WebDriverConfig;
+import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.Configuration;
 import org.springframework.context.annotation.Import;
 
+import java.util.concurrent.ThreadPoolExecutor;
+
 @Configuration
 @Import({WebDriverConfig.class, ThreadPoolConfig.class})
 public class AppConfig {
+
+    @Bean
+    public ObjectMapper objectMapper() {
+        return new ObjectMapper();
+    }
```

Comment:
Can be deleted since ObjectMapper is autoconfigured by Spring Boot & we don't need additional logic for it right now.